src/Tools/BuildBoss/ProjectCheckerUtil.cs

```diff
@@ -236,6 +237,36 @@ private bool CheckPackageReferences(TextWriter textWriter)
             return allGood;
         }
 
+        private bool CheckInternalsVisibleTo(TextWriter textWriter)
+        {
+            var allGood = true;
+            foreach (var internalsVisibleTo in _projectUtil.GetInternalsVisibleTo())
+            {
+                if (string.Equals(internalsVisibleTo.LoadsWithinVisualStudio, "false", StringComparison.OrdinalIgnoreCase))
+                {
+                    // IVTs explicitly declared with LoadsWithinVisualStudio="false" are allowed
```

Comment:
This is for the Xamarin case?

Reply:
It's for VS for Mac and MonoDevelop (intersects with Xamarin, but some Xamarin assemblies also load in VS and will use external access assemblies).